WordPress/Classes/ViewRelated/Jetpack/Jetpack Scan/View Models/JetpackScanThreatViewModel.swift

```diff
@@ -309,8 +311,128 @@ struct JetpackScanThreatViewModel {
     }
 }
 
+extension JetpackScanThreatViewModel {
+
+    private static func attributedString(for context: JetpackThreatContext?) -> NSAttributedString? {
+        guard let context = context else {
+            return nil
+        }
+
+        let lineFont = UIFont.monospacedSystemFont(ofSize: 13, weight: .semibold)
+        let numberFont = UIFont.monospacedSystemFont(ofSize: 13, weight: .bold)
+        let attrString = NSMutableAttributedString()
+
+        let biggestLen = String(context.lines.last?.lineNumber ?? 0).count
+
+        let longestLine = context.lines.sorted { $0.contents.count > $1.contents.count}.first!
+        let longestLen = longestLine.contents.count
+
+        let paragraph = NSMutableParagraphStyle()
+
+        for line in context.lines {
+
+            // Number attributed string
+
+            var numberStr = String(line.lineNumber)
+            let numberPadding = "".padding(toLength: biggestLen - numberStr.count,
+                                           withPad: Constants.noBreakSpace, startingAt: 0)
+            numberStr = numberPadding + numberStr
+            let numberAttr = NSMutableAttributedString(string: numberStr)
+
+            // Contents attributed string
+
+            let contents = line.contents
+            let contentsPadding = "".padding(toLength: longestLen - contents.count,
+                                             withPad: Constants.noBreakSpace, startingAt: 0)
+
+            let padding = " "
```

Comment:
I forgot what this was for, we can probably rename it to something like this or .. ?

We should also move this to Constants

```suggestion
            let columnSpacer = " "
```